apps/site/assets/ts/schedule/__tests__/LineDiagramTest.tsx

```diff
@@ -63,7 +63,8 @@ export const lineDiagram = [
       ],
       closed_stop_info: null,
       branch: null
-    }
+    },
+    alerts: []
```

Comment:
We should add an example that has an alert. To get some alert data, you can grab the data from React DevTools (which I can show you in person), or you could hit the line api in your browser.